CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt

```diff
@@ -148,10 +148,10 @@ Kcira, Dorian - Caltech
 Kersevan, Borut Paul - ULjubljana
 Kirby, Michael - Fermilab (3)
 Klimentov, Alexei - BNL
-Klute, Markus - MIT
+Klute, Markus - UMass
 Komarov, Ilya - INFNTRIESTE (14)
 Konstantinov, Dmitri - Protvino
-Koppenburg, Patrick - NIKHEF
+Koppenburg, Patrick - NIKHEF & VU
```

Comment:
Gerhard Raven is indeed Nikhef and VU but Patrick is only employed by Nikhef. I've just checked this on the LHCb authors list to be sure.